Warnings in Point_set_shape_detection_3

The column "Debian-Testing" of recent testsuite show two warnings in point set shape detection and related line:
```
[...]/include/CGAL/Shape_detection_3/Cone.h:428:12: warning: variable 'radPerDist' set but not used [-Wunused-but-set-variable]
[...]/include/CGAL/Shape_detection_3/Octree.h:666:14: warning: variable 'v1' set but not used [-Wunused-but-set-variable]
```
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.11-Ic-12.shtml#Point_set_3_Examples

I saw the warnings in CGAL-4.11-Ic-12, but I think the bugs are in `master`.

If you see that the bug in in `releases/CGAL-4.10-branch` too, please create the PR on top of it.

--- a/Point_set_shape_detection_3/include/CGAL/Shape_detection_3/Cone.h
+++ b/Point_set_shape_detection_3/include/CGAL/Shape_detection_3/Cone.h
@@ -425,8 +425,7 @@ namespace CGAL {
         Vector_3 d = this->constr_vec(m_apex, this->point(indices[0]));
         FT v = this->scalar_pdct(d, m_axis) / m_cos_ang;
         FT phi = atan2(this->scalar_pdct(d, d2), this->scalar_pdct(d, d1));
-        FT radPerDist = -m_neg_sin_ang * v;
-        FT u = FT(phi + CGAL_PI);// * radPerDist;
+        FT u = FT(phi + CGAL_PI);
         FT avg_v = v;
 
         min[0] = max[0] = u;
@@ -436,8 +435,7 @@ namespace CGAL {
           d = this->constr_vec(m_apex, this->point(indices[i]));
           v = this->scalar_pdct(d, m_axis) / m_cos_ang;
           phi = atan2(this->scalar_pdct(d, d2), this->scalar_pdct(d, d1));
-          radPerDist = -m_neg_sin_ang * v;
-          u = FT(phi + CGAL_PI);// * radPerDist;
+          u = FT(phi + CGAL_PI);
 
           min[0] = (std::min<FT>)(min[0], u);
           max[0] = (std::max<FT>)(max[0], u);

--- a/Point_set_shape_detection_3/include/CGAL/Shape_detection_3/Octree.h
+++ b/Point_set_shape_detection_3/include/CGAL/Shape_detection_3/Octree.h
@@ -662,8 +662,6 @@ namespace CGAL {
 
         while(first < last) {
           // find first above threshold
-          Point_3 p1 = get(m_point_pmap, *this->at(first));
-          FT v1 = get_coord(p1, static_cast<unsigned int>(dimension));
           while (get_coord(
                    get(m_point_pmap, *this->at(first)),
                    static_cast<unsigned int>(dimension)) < threshold
@@ -680,8 +678,6 @@ namespace CGAL {
           }
 
           // find last below threshold
-          p1 = get(m_point_pmap, *this->at(last));
-          v1 = get_coord(p1, static_cast<unsigned int>(dimension));
           while (get_coord(
                    get(m_point_pmap, *this->at(last)),
                    static_cast<unsigned int>(dimension)) >= threshold
@@ -698,10 +694,6 @@ namespace CGAL {
           }
 
           this->swap(first, last);
-          p1 = get(m_point_pmap, *this->at(first));
-          v1 = get_coord(p1, static_cast<unsigned int>(dimension));
-          p1 = get(m_point_pmap, *this->at(last));
-          v1 = get_coord(p1, static_cast<unsigned int>(dimension));
           first++;
           last--;
         }

